scripts/update-distribution-scripts.sh

```diff
@@ -38,15 +42,19 @@ sed -i -E "/distribution-scripts-version:/{n;n;n;s/default: \".*\"/default: \"$r
 git add .circleci/config.yml
 
 message="Updates \`distribution-scripts\` dependency to https://github.com/crystal-lang/distribution-scripts/commit/$reference"
-log=$($GIT_DS log $old_reference..$reference --format="%s" | sed "s/.*(/\* crystal-lang\/distribution-scripts/;s/.$//")
-message="$message\n\nThis includes the following changes:\n\n$log"
+log=$($GIT_DS log $old_reference..$reference --format="%s" | sed "s/.*(/crystal-lang\/distribution-scripts/;s/^/* /;s/.$//")
+message=$(printf "$message\n\nThis includes the following changes:\n\n$log")
```

Comment:
Will this break if the message contains `%`?

Reply:
Yes. The answer is yes 😅
```suggestion
message=$(printf "%s" "$message\n\nThis includes the following changes:\n\n$log")
```

Reply:
```suggestion
message=$(printf "%s\n\nThis includes the following changes:\n\n%s" "$message" "$log")
```